x/distribution/types/delegator.go

```diff
@@ -14,7 +14,7 @@ import (
 type DelegatorStartingInfo struct {
 	PreviousPeriod uint64  `json:"previous_period"` // period at which the delegation should withdraw starting from
 	Stake          sdk.Dec `json:"stake"`           // amount of staking token delegated
-	Height         uint64  `json:"height"`          // height at which delegation was created
```

Comment:
I believe this is API breaking, but otherwise it could be a point of confusion if two fields being returned both say "height"